frontend/lib/components/TextEditor/components/Toolbar/index.ts

```diff
@@ -0,0 +1 @@
+export * from "./Toolbar"
```

Comment:
we decided no to use index.ts for now on our fodlers, to have a clearer codebase
There are still some places with index.ts, but we will remove them

Same comment fot other index.ts :)